### DISCUSSION TITLE
Pre-commit notes in CONTRIBUTING instead of README?

### DISCUSSION THREAD

**Discussion by abravalheri** [pyscaffold/pyscaffold#523] [Ideas]
Now that we have a full-blow `CONTRIBUTING.rst` template, does it make sense to have:

https://github.com/pyscaffold/pyscaffold/blob/bb9ff6df83aa98b875addd1f8eacbd572961e79d/src/pyscaffold/extensions/pre_commit.py#L48-L60

on README? Maybe this information should be added to CONTRIBUTING instead...

**Comment by FlorianWilhelm** [comment]
I think this makes total sense to have all the details in CONTRIBUTING. Maybe it could only be mentioned in the README like "If you consider contributing to this project, find all details in CONTRIBUTING...".

**Comment by julie777** [comment]
IMHO

I think that README should be short and general.

It can reference other sources such as CONTRIBUTING, the LICENSE or the documentation. But, let's remember that the README is the github landing spot. I'm not sure, but it might also be used for pypi. In any case, most readers of the README aren't interested in contributing.
